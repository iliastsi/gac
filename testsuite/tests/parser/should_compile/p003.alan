hello() : proc
    c : int;	
	foo(f1:int, f2:reference byte) : proc
	foo2 : byte;
	{}
{
  if (false) {
    writeString("What??\n");
  } else
    writeString("Hello world!\n");

	while (false) {
	c = 1+ 10 *50 - 2 /3;
    writeString("What??\n");
	}
}
