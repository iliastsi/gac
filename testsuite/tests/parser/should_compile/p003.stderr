==================== Parser ====================
hello() : proc
  c : int;
  foo(f1 :  int, f2 : reference byte) : proc
    foo2 : byte;
  { -- foo
  } -- foo
{ -- hello
  if(false) {
    writeString("What??\n");
  } else
    writeString("Hello world!\n");
  while(false) {
    c = 1 + 10 * 50 - 2 / 3;
    writeString("What??\n");
  }
} -- hello


